app/controllers/alaveteli_pro/plans_controller.rb

```diff
@@ -4,7 +4,8 @@ class AlaveteliPro::PlansController < ApplicationController
   before_filter :authenticate
 
   def show
-    @plan = Stripe::Plan.retrieve(params[:id])
+    stripe_plan = Stripe::Plan.retrieve(params[:id])
```

Comment:
this is going to be a bit clunky for reuse in the subscription code as we don't fetch the plan directly, it's access through `customer.subscription.plan`, having think